erpnext/projects/doctype/timesheet/timesheet.py

```diff
@@ -62,6 +59,43 @@ def calculate_percentage_billed(self):
 		if self.total_billed_amount > 0 and self.total_billable_amount > 0:
 			self.per_billed = (self.total_billed_amount * 100) / self.total_billable_amount
 
+	def validate_overtime(self):
+		total_overtime_hours= 0
+		overtime_type = None
+		for data in self.time_logs:
+			overtime_type = data.overtime_type
+			if data.is_overtime:
+				if frappe.db.get_single_value("Payroll Settings", "overtime_based_on") == "Timesheet":
+					if not self.employee:
+						frappe.throw(_("Select Employee, if applicable for overtime"))
+
+					if not data.overtime_type:
+						frappe.throw(_("Define Overtime Type for Employee {0}").format(self.employee))
+
+					if data.overtime_on:
+						if data.overtime_on <= data.from_time or data.overtime_on >= data.to_time:
```

Comment:
![image](https://user-images.githubusercontent.com/24353136/126978155-2bb170f0-67eb-42d7-a4e8-585b6ac8941c.png)

**Overtime On** field type is **Date** and it is being compared with **Time** fields. Not allowing me to save the timesheet. Also for validations involving date/time, it should be shown in user format

![image](https://user-images.githubusercontent.com/24353136/126978393-274ce404-c7dc-44f4-bd9f-fa0615d743c0.png)

Reply:
Done